python/ql/test/query-tests/Statements/asserts/assert.py

```diff
@@ -16,8 +16,8 @@ def assert_tuple(x, y):
     assert ()
     assert (x, y)
 
-
-
+import subprocess
+assert subprocess.call(['run-backup']) == 0 # TODO: FN
```

Comment:
That was probably a mistake. I don't recall the commit history, but I guess that at that point it wasn't flagged, and when the `TODO` comment was removed, it was added to the `.expected` file -- at least that would have been my intention :smile:

I can go back and fix things if you feel strongly about this, but since it is not included in the final files of this PR, I'm inclined to do nothing ¯\\\_(ツ)\_/¯

Reply:
Oh, now I found a way to see the commit your comment was attached to. that _does_ seem like a mistake

Reply:
Ah, the dangers of reviewing commit-by-commit. I apologise for the confusion. (It would be nice if the UI would inform me that that file gets changed again later on. Hmm...)